PokemonGo.RocketAPI/Helpers/Utils.cs

```diff
@@ -9,5 +12,37 @@ public static ulong FloatAsUlong(double value)
             var bytes = BitConverter.GetBytes(value);
             return BitConverter.ToUInt64(bytes, 0);
         }
+
+        public static uint GenerateLocation1(byte[] authTicket, double lat, double lng, double alt)
+        {
+            var seed = BitConverter.ToUInt32(new xxHash(32, 0x1B845238).ComputeHash(authTicket), 0);
+            var xxh32 = new xxHash(32, seed);
+
+            var locationBytes = new List<byte>();
+            locationBytes.AddRange(BitConverter.GetBytes(lat).Reverse());
+            locationBytes.AddRange(BitConverter.GetBytes(lng).Reverse());
+            locationBytes.AddRange(BitConverter.GetBytes(alt).Reverse());
```

Comment:
Ah, okay I though GetBytes automatically converts that back, but so this makes sense. Thank you :)